res/layout/registration_activity.xml

```diff
@@ -151,7 +151,7 @@
             app:layout_constraintStart_toEndOf="@id/advanced_icon"
             app:layout_constraintTop_toBottomOf="@id/no_servers_hint" />
 
-        <com.google.android.material.checkbox.MaterialCheckBox
+        <CheckBox
```

Comment:
just did the same as on the "confirm autodeletion of messages" dialog where the checkbox is nice in light/dark modes: https://github.com/deltachat/deltachat-android/blob/main/res/layout/dialog_with_checkbox.xml#L25